modules/nw-dns-loglevel.adoc

```diff
@@ -0,0 +1,45 @@
+// Module included in the following assemblies:
+// * networking/dns/nw-operator.adoc
+
+:_content-type: PROCEDURE
+[id="nw-dns-loglevel_{context}"]
+= Set CoreDNS logLevel
+
+Valid values for CoreDNS logLevel are `Normal`, `Debug`, and `Trace`.
+
+.Procedure
+Default logLevel is `Normal`.
+
+Set logLevel to `Debug`:
```

Comment:
* To set `logLevel` to `Debug`, enter the following command:
+
[source,terminal]
----
$ oc patch dnses.operator.openshift.io/default -p '{"spec":{"logLevel":"Debug"}}' --type=merge
----

* To set `logLevel` to `Trace`, enter the following command:
+ 
[source,terminal]
----
$ oc patch dnses.operator.openshift.io/default -p '{"spec":{"logLevel":"Trace"}}' --type=merge
----



This adds the the bullet point for the procedures (since they're not an ordered process) and the "enter the following command" instruction. There's more info here: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#code-blocks-command-syntax-and-example-output

Reply:
Thank you so much!